pom.xml

```diff
@@ -740,6 +741,11 @@
         <version>${org.owasp.antisamy.version}</version>
       </dependency>
       <dependency>
+        <groupId>com.googlecode.owasp-java-html-sanitizer</groupId>
```

Comment:
The dependency list is sorted by alphabetical order of the groupId ( @mgreau ;-) )